kyaml/kio/filters/fmtr_test.go

```diff
@@ -13,10 +13,195 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/kustomize/kyaml/kio"
 	. "sigs.k8s.io/kustomize/kyaml/kio/filters"
 	"sigs.k8s.io/kustomize/kyaml/kio/filters/testyaml"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
 )
 
+func TestFormatInput_FixYaml1_1Compatibility(t *testing.T) {
```

Comment:
I see these are more like end to end tests. Aren't unit tests needed ?

Reply:
Are your concerns with coverage or test run time (or something else)?

Reply:
I am talking more about modular(unit) tests coverage. For example there is no independent unit test with name **testFmtNode** which mocks all the inputs for that particular method and verifies the output of it in modular fashion. I agree that it is covered as part of this broader scope test but it makes the method to be tightly coupled other methods to be tested.

Reply:
Resolved offline